### PR TITLE
Prepare e-h-nb-1.0.0-alpha.0 release

### PR DESCRIPTION
**Pull request by eldruin**

### CODE REVIEW

File: embedded-hal-nb/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+...
+
+## [v0.1.0-alpha.0] - 2022-09-27
```
Comment (Dirbaio) [inline embedded-hal-nb/CHANGELOG.md:12]: v0.1.0 or v1.0.0?

Comment (eldruin) [reply embedded-hal-nb/CHANGELOG.md]: Nice catch, should be fixed now.

**Review by Dirbaio — APPROVED** [review]
:rocket: !!!

bors r+

**Comment by burrbull** [comment]
e-h-1.0-alpha.9 ?

**Comment by eldruin** [comment]
@burrbull there has certainly be enough changes. Let's do that after an `embedded-can` release: https://github.com/rust-embedded/embedded-hal/pull/411

**Pull request merged**